caffe2/quantization/server/conv_dnnlowp_op.cc

```diff
@@ -1223,7 +1223,9 @@ void ConvDNNLowPOp<T, ReluFused>::ConvNHWCCore_(
             out_qparams_.zero_point,
             in_qparams_[INPUT].zero_point,
             filter_zero_points_.data(),
-            row_offsets_.data() + tid * row_offset_size_per_thread,
+            filter_zero_points_[0]
```

Comment:
if this is for symmetric quantization, is there any other way than to check the numeric value of the data?

Reply:
Sorry can you elaborate more? What do you mean by numeric value of the data?

Reply:
I understood this as checking to see if the zero_point is 0 or not? If that is correct, I was wondering if there should be no storage at all. It's just nit, not a big issue.

Reply:
You mean if zero_point is 0 then filter_zero_points_ should be empty? It's hard to do that in the current design because I believe we're also using the size of filter_zero_points_ to tell if we're using group-wise quantization.